src/components/Edit.js

```diff
@@ -0,0 +1,188 @@
+import Dates from "./Dates";
+import styled from "styled-components";
+
+import { useState } from "react";
+
+export default function Edit(props) {
+  const { list, setList } = props;
+  const [scheduleChecked, setScheduleChecked] = useState(true);
+  const [doneChecked, setDoneChecked] = useState(true);
+
+  function handleTrash(id) {
+    const updatedList = list.filter((data) => data.id !== id);
+    setList(updatedList);
+    localStorage.setItem("testify", JSON.stringify(updatedList));
+  }
+
+  function handleStatus(id) {
+    const updatedList = list.map((data) => {
+      if (data.id === id) {
+        return {
+          ...data,
+          schedule: !data.schedule,
+        };
+      }
+      return data;
+    });
```

Comment:
사소하지만 
```suggestion
    const updatedList = list.map((data) =>
      data.id === id ? { ...data, schedule: !data.schedule } : data
    );
```
이렇게 간단하게 처리해도 될 것 같아요 !!